src/pint/models/timing_model.py

```diff
@@ -4072,7 +4072,15 @@ class UnknownParameter(TimingModelError):
 class UnknownBinaryModel(TimingModelError):
     """Signal that the par file requested a binary model no in PINT."""
```

Comment:
no in --> is not in

Reply:
Ha, good catch. I did not write that actually, my git lens blames Jing. But I'm updating it 👍